requirements.txt

```diff
@@ -4,5 +4,5 @@ scikit-optimize
 jsonschema
 grpcio
 click
-protobuf==3.19.4
-plotly
\ No newline at end of file
+protobuf==4.21.8
```

Comment:
@kusumachalasani - Do we need to specify the version here? Without explicitly specifying version, doesn't it install the latest protobuf ?

Reply:
Its not updating if there is a previous version of protobuf.

Reply:
This is a question about dependency versioning and repeatable builds.  by not versioning dependencies, builds are not repeatable at different points in time, as newer versions are automatically resolved. This can introduce issues due to incompatibilities. There is currently an inconsistency in the python requirements file, between explicit and latest versions of dependencies.  I would suggest picking a standard for the project and applying it consistently.  My inclination would be to explicitly state versions, and then use something like dependabot to explicitly manage dependency versions

Reply:
@johnaohara Yes. we do accept that it should be consistent and explicitly state versions for all the dependencies. We shall take this as part of next PR.